include/ccf/base_endpoint_registry.h

```diff
@@ -3,10 +3,10 @@
 #pragma once
 
 #include "ccf/crypto/pem.h"
+#include "ccf/ds/quote_info.h"
```

Comment:
The move to ds is a little surprising, given that we were trying to keep ds dependency-free, as I understand it.